util/src/main/java/io/kubernetes/client/util/Config.java

```diff
@@ -137,4 +138,36 @@ public static ApiClient fromConfig(Reader input) {
 
         return client;
     }
+
+    /**
+     * Easy client creation, follows this plan
+     * <ul>
+     *   <li>If $KUBECONFIG is defined, use that config file
+     *   <li>If $HOME/.kube/confg can be found, use that.
+     *   <li>If the in-cluster service account can be found, assume in cluster config.
+     *   <li>Default to localhost:8080 as a last resort.
+     * <ul>
+     * 
+     * @returns The best APIClient given the previously described rules
+     */
+    public static ApiClient defaultClient() throws IOException {
+        String kubeConfig = System.getenv("KUBECONFIG");
+        if (kubeConfig != null) {
+            return fromConfig(new FileReader(kubeConfig));
+        }
+        String homeDir = System.getenv("HOME");
+        if (homeDir != null) {
+            File homeConfig = new File(new File(new File(homeDir), ".kube"), "config");
```

Comment:
Can we extract constant for all of these strings? ".kube", "config" "HOME", "KUBECONFIG". I suggest we use something like "~/.kube.config" in constant and expand ~.